packages/sst/src/constructs/SsrSite.ts

```diff
@@ -1067,6 +1077,11 @@ export class SsrSite extends Construct implements SSTConstruct {
     }
   }
 
+  protected getS3ContentReplaceValues(): SsrSiteReplaceProps[] {
+    const replaceValues: SsrSiteReplaceProps[] = this.props.replaceValues || [];
+    return replaceValues;
+  }
```

Comment:
Default for base class is to do nothing. Recognizing now that I could probably just remove this and do any transforms I need in `NextjsSite` constructor